app/src/main/java/io/gnosis/safe/ui/safe/add/AddSafeNameViewModel.kt

```diff
@@ -31,7 +33,8 @@ class AddSafeNameViewModel
             runCatching {
                 val safe = Safe(address, localName.trim())
                 safeRepository.saveSafe(safe)
-                notificationRepository.registerSafe(safe.address)
+                notificationRepository.registerSafe(safe)
+                notificationManager.createNotificationChannelGroup(safe)
```

Comment:
the channel should also be removed when removing a safe, right?

Reply:
`deleteNotificationChannelGroup()` removes the all groups channels and the group itself. 
```
    fun deleteNotificationChannelGroup(safe: Safe) {
        val subChannels = ChannelType.values()

        subChannels.forEach { subChannel ->
            notificationManager.deleteNotificationChannel(safe.address.asEthereumAddressChecksumString() + "." + context.getString(subChannel.resId))
        }
        notificationManager.deleteNotificationChannelGroup(safe.address.asEthereumAddressChecksumString())
    }
´´´

Called in `SafeSettingsViewModel.removeSafe()`